src/blindrsa.ts

```diff
@@ -26,8 +26,8 @@ export class BlindRSA {
     private static readonly NAME = 'RSA-PSS';
 
     constructor(
-        private readonly hash: string,
-        private readonly saltLength: number,
+        public readonly hash: string,
```

Comment:
flagging for info: this same line is modified in #7 . It's going to create merge conflict